### ISSUE TITLE
Deafault admin user

### ISSUE THREAD

**Issue by fradaloisio** [preston/bittorious#5]
There isn't a default admin user.

**Comment by preston** [comment]
Correct. :) I think sometime around BitTorious 2.x or 3.x the system was refactored to allow for multiple administrators once configured. Out of the box, though, assuming you seed the database, there *is* a default admin with login credentials shown here: https://github.com/preston/bittorious/blob/master/db/seeds.rb
